src/ci/bin/test.inc.sh

```diff
@@ -120,7 +120,7 @@ cromwell::private::create_build_variables() {
     # circuiting logic for pull request builds
     if [[ "${TRAVIS_EVENT_TYPE:-unset}" != "pull_request" ]]; then
         CROMWELL_BUILD_ONLY_DOCS_CHANGED=false
-    elif git diff --name-only "origin/${TRAVIS_BRANCH}" 2>&1 | grep -q --invert-match ^docs/; then
```

Comment:
Hmm, AFAIR @mcovarr just recently added that `origin/` part here. Why do we need to remove it again?

Reply:
Not deliberate, I copy/pasted and then modified the content from your original PR instead of the current version of this line.

Reply:
Should be resolved now